src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/IHtmlGenerator.cs

```diff
@@ -157,6 +157,7 @@ public interface IHtmlGenerator
             ModelExplorer modelExplorer,
             string expression,
             object value,
+            string format,
```

Comment:
If we can't add a method to this interface (even though we don't expect another implementation), about all that comes to mind is duplicating the few lines of formatting code in the `<input/>` tag helper. @Haplois any other ideas?